tests/cloner_test.go

```diff
@@ -1135,6 +1135,90 @@ var _ = Describe("all clone tests", func() {
 		})
 
 	})
+
+	var _ = Describe("Preallocation", func() {
+		f := framework.NewFramework(namespacePrefix)
+
+		var sourcePvc *v1.PersistentVolumeClaim
+		var targetPvc *v1.PersistentVolumeClaim
+		var cdiCr cdiv1.CDI
+		var cdiCrSpec *cdiv1.CDISpec
+
+		BeforeEach(func() {
+			By("[BeforeEach] Saving CDI CR spec")
+			crList, err := f.CdiClient.CdiV1beta1().CDIs().List(context.TODO(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(len(crList.Items)).To(Equal(1))
+
+			cdiCrSpec = crList.Items[0].Spec.DeepCopy()
+			cdiCr = crList.Items[0]
+
+			By("[BeforeEach] Forcing Host Assisted cloning")
+			var cloneStrategy cdiv1.CDICloneStrategy = cdiv1.CloneStrategyHostAssisted
+			cdiCr.Spec.CloneStrategyOverride = &cloneStrategy
+			_, err = f.CdiClient.CdiV1beta1().CDIs().Update(context.TODO(), &cdiCr, metav1.UpdateOptions{})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
You probably want to ensure that the clone strategy is set in a fresh object before going onto the test, sometimes it takes a few seconds to update, and during that time you don't want your test to run.

Reply:
Done